metrics/prometheus.go

```diff
@@ -1725,6 +1725,13 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 			},
 		}...)
 	}
+	var filtered []containerMetric
```

Comment:
Besides Prometheus there are other ways of exposing metrics. This logic would have to be added to all of them.

Reply:
Hi,iwankgb! I think it makes sense.But I don't know the specific ways of exposing metrics which cadvisor has used.Besides  it seems the param "disable_metrics" also only supports Prometheus.So could you tell me which ways of exposing metrics  we should support? many thanks!

Reply:
I was not aware that `disable_metrics` is limited to Prometheus.